spec.emu

```diff
@@ -190,16 +190,17 @@ contributors: Ujjwal Sharma, Younies Mahmoud
       <h1>
         GetDurationUnitOptions (
           _unit_: a String,
-          _options_: an ECMAScript language value,
+          _options_: an Object,
           _baseStyle_: a String,
-          _stylesList_: a List of String,
+          _stylesList_: a List of Strings,
           _digitalBase_: a String,
           _prevStyle_: a String,
+          _hoursDigits_: an integer,
         ): either a normal completion containing a Record with [[Style]] and [[Display]] fields, or an abrupt completion
       </h1>
       <dl class="header">
         <dt>description</dt>
-        <dd>It extracts the relevant options for any given _unit_ from an options bag and returns them as a Record.</dd>
+        <dd>It extracts the relevant options for any given _unit_ from an Object and returns them as a Record.</dd>
```

Comment:
An option could also be
```suggestion
        <dd>It extracts the relevant options for any given _unit_ from _options_ and returns them as a Record.</dd>
```

Reply:
there doesn't appear to be prior art for that in either 402 or 262